### ISSUE TITLE
ONNX model Release

### ISSUE THREAD

**Issue by shubhammore1310** [ysharma3501/NovaSR#15]
Hi Team, 
Can you release one ONNX model ? 
I want to implement C++ inference for this project.
Model I am exporting is not proper.

_import torch
from NovaSR import FastSR
ckpt_path = 'pytorch_model.bin'
upsampler = FastSR(ckpt_path=ckpt_path, half=False)
upsampler.model.to('cpu').eval()
dummy_input = torch.randn(1, 1, 66594, dtype=torch.float32)  # Explicit float32
torch.onnx.export(
    upsampler.model,
    dummy_input,
    "novasr_fixed.onnx",
    export_params=True,
    opset_version=17,
    do_constant_folding=False,  # CRITICAL: Disable folding for accuracy
    input_names=['input'],
    output_names=['output'],
    dynamic_axes={
        'input': {0: 'batch_size', 2: 'time_steps'},   # Batch + time dynamic
        'output': {0: 'batch_size', 2: 'output_time_steps'}
    }
)
print("Fixed export complete.")_

**Comment by hlevring** [comment]
Yeah an onnx model would be great. @shubhammore1310 did you manage to get it converted?

**Comment by shubhammore1310** [comment]
I exported the model with code above, but output is worse than 16khz. 
I calculated the **MSE** of audio from pytorch model and onnx model, 
_MSE  : 0.0019201676
MAE  : 0.0301390671
RMSE : 0.0438197176_
I'll create the fork repo and share the model I exported.
For the audio quality to be similar, MSE values should be less than 1e-5 or 1e-5.
